xray/src/bin/web_viewer.rs

```diff
@@ -0,0 +1,247 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extern crate cgmath;
+#[macro_use]
+extern crate clap;
+extern crate collision;
+#[macro_use]
+extern crate iron;
+extern crate protobuf;
+extern crate quadtree;
+extern crate router;
+extern crate serde;
+extern crate serde_json;
+extern crate urlencoded;
+extern crate xray;
+
+#[macro_use]
+extern crate serde_derive;
+
+use cgmath::{Matrix4, Point3};
+use collision::{Aabb3, Frustum, Relation};
+use iron::mime::Mime;
+use iron::prelude::*;
+use quadtree::{ChildIndex, Node};
+use router::Router;
+use std::fs;
+use std::io::Read;
+use std::path::PathBuf;
+use std::sync::Arc;
+use urlencoded::UrlEncodedQuery;
+use xray::Meta;
+
+const INDEX_HTML: &'static str = include_str!("../../client/index.html");
```

Comment:
The octree web_viewer and the quadtree web_viewer put their in the same location.

Reply:
Actually that wasn't the case. We misread the code yesterday.